Configuration/Skimming/python/Skims_PbPb_cff.py

```diff
@@ -50,3 +50,16 @@
     )
 
 #####################      
+
+from Configuration.Skimming.PbPb_ZMuSkimMuonDPG_cff import *
+ZMuSkimPath = cms.Path( diMuonSelSeq )
+SKIMStreamZMu = cms.FilteredStream(
```

Comment:
@mandrenguyen this must become SKIMStreamPbPbZMu , that is the reason for the test failure